collections/requirements.yml

```diff
@@ -0,0 +1,5 @@
+---
+collections:
+  - name: community.general
+  - name: ansible.posix
+  - name: awx.awx
```

Comment:
Not sure if QE will be OK with merging always having requirements file cc @Zokormazo ...
In the past, they've kept branches open and referenced them specifically when wanting to use the requirements.

Maybe we could merge the 2 playbooks seperate so they are always on main branch, and keep requirements on a branch. You can see there are other PRs marked [Do Not Merge] or something like that

Reply:
I can remove it, and create new branch with this file, so depends on the branch used , it will have or not this file.